pkgs/development/libraries/science/networking/ns-3/default.nix

```diff
@@ -79,7 +78,7 @@ stdenv.mkDerivation rec {
     sqlite.dev
     gsl
     boost
-    root
+    root # provides cppyy
```

Comment:
what is cppyy ?

Reply:
this
https://github.com/wlav/cppyy
or
https://github.com/root-project/root/tree/master/bindings/pyroot/cppyy/cppyy